config/options.go

```diff
@@ -41,6 +41,7 @@ var (
 
     // Server
     WEB_PORT = MustNewIntOption("web.port", 8080, "The port for the web interface to serve on.");
+    WEB_MAX_FILE_SIZE = MustNewIntOption("web.maxsize", 4 << 20, "The maximum allowed file size submitted via POST request.");
```

Comment:
For options that users set, we want to make them a bit more readable.
Make this value in KB.

You can also make this a float so you can specify smaller values (like in tests).

Reply:
My test no longer passes when I make this a float.  I will look further into it.

Reply:
This comment is probably not necessary.
Remember that comments on my comments (replies to the thread) should indicate a discussion that I need to take part in.
No need to update me on every step, only when you need feedback or questions.
Otherwise, you can leave the comments until you are ready to resolve or continue the discussion.